proto/viam/common/v1/common.proto

```diff
@@ -113,8 +114,8 @@ message PointCloudObject {
 }
 
 message GeoPoint {
-  double latitude = 1;
-  double longitude = 2;
+  double latitude = 1 [(tagger.v1.tags) = "bson:\"latitude\""];
+  double longitude = 2 [(tagger.v1.tags) = "bson:\"longitude\""];
```

Comment:
What happens if these ever change?

Reply:
@randhid Had a few discussions with some Core peeps and we're on the fence about whether to follow through with adding this validation altogether - it makes existing triggers reliant on the latest API types which is an issue for someone using a robot whose data format is out of sync with those latest types...I will circle back here if there are is need to add these tags, but for the meantime we will hold off on this addition

Thank you for hopping on this last week, sorry to have wasted any time

Reply:
no worries! happy to jump in when/if the time comes.